### PR TITLE
PR: New scrolls for the old merit spell

### PR DESCRIPTION
**Pull request by m241dan**
Added the "new" scrolls for the old merit spells.

Some item subids are incorrect

### CODE REVIEW

File: scripts/globals/items/scroll_of_burst_ii.lua [hunk 1]
```diff
@@ -0,0 +1,13 @@
+-----------------------------------------
+-- ID: 4821
+-- Scroll of Burst
+-- Teaches the black magic Burst
```
Comment (kaincenteno) [inline scripts/globals/items/scroll_of_burst_ii.lua]: Change to Burst II

File: scripts/globals/items/scroll_of_flare_ii.lua [hunk 2]
```diff
@@ -0,0 +1,13 @@
+-----------------------------------------
+-- ID: 4813
+-- Scroll of Flare
+-- Teaches the black magic Flare
```
Comment (kaincenteno) [inline scripts/globals/items/scroll_of_flare_ii.lua]: Change to Flare II

File: scripts/globals/items/scroll_of_flood_ii.lua [hunk 3]
```diff
@@ -0,0 +1,13 @@
+-----------------------------------------
+-- ID: 4823
+-- Scroll of Flood
+-- Teaches the black magic Flood
```
Comment (kaincenteno) [inline scripts/globals/items/scroll_of_flood_ii.lua]: Change to Flood II

File: scripts/globals/items/scroll_of_freeze_ii.lua [hunk 4]
```diff
@@ -0,0 +1,13 @@
+-----------------------------------------
+-- ID: 4815
+-- Scroll of Freeze
+-- Teaches the black magic Freeze
```
Comment (kaincenteno) [inline scripts/globals/items/scroll_of_freeze_ii.lua]: Change to Freeze II

File: scripts/globals/items/scroll_of_quake_ii.lua [hunk 5]
```diff
@@ -0,0 +1,13 @@
+-----------------------------------------
+-- ID: 4819
+-- Scroll of Quake
+-- Teaches the black magic Quake
```
Comment (kaincenteno) [inline scripts/globals/items/scroll_of_quake_ii.lua]: Change to Quake II

File: scripts/globals/items/scroll_of_suiton_san.lua [hunk 6]
```diff
@@ -0,0 +1,13 @@
+-----------------------------------------
+-- ID: 4945
+-- Scroll of Suiton: Ni
+-- Teaches the ninjutsu Suiton: Ni
```
Comment (kaincenteno) [inline scripts/globals/items/scroll_of_suiton_san.lua]: Update to San (verify the spell numbers are correct?)

File: scripts/globals/items/scroll_of_tornado_ii.lua [hunk 7]
```diff
@@ -0,0 +1,13 @@
+-----------------------------------------
+-- ID: 4817
+-- Scroll of Tornado
+-- Teaches the black magic Tornado
```
Comment (kaincenteno) [inline scripts/globals/items/scroll_of_tornado_ii.lua]: Change to Tornado II

File: scripts/zones/Lower_Jeuno/npcs/Hasim.lua [hunk 8]
```diff
@@ -72,6 +72,11 @@ function onTrigger(player,npc)
         4647,  20000,    -- Scroll of Banishga II
         4737, 119240,    -- Scroll of Protecra V
         4742, 124540,    -- Scroll of Shellra V
+        4633, 139135,    -- Scroll of Dia III
+        6569, 139135,    -- Scroll of Slow II
+        6570, 139135,    -- Scroll of Paralyze II
+        6571, 139135,    -- Scroll of Phalanx II
+
```
Comment (kaincenteno) [inline scripts/zones/Lower_Jeuno/npcs/Hasim.lua]: remove empty line

**Comment by kaincenteno** [comment]
Awesome contribution dood, sorry for the many comments.

**Comment by m241dan** [comment]
> Awesome contribution dood, sorry for the many comments.

No worries, I should have noticed that stuff... managed to get the comments right for all the others... was pretty tired at this point though.

File: sql/item_basic.sql [hunk 9]
```diff
@@ -4375,6 +4375,7 @@ INSERT INTO `item_basic` VALUES (4629,21,'scroll_of_holy','holy',1,1676,28,0,216
 INSERT INTO `item_basic` VALUES (4630,22,'scroll_of_holy_ii','holy_ii',1,34444,28,0,0);
 INSERT INTO `item_basic` VALUES (4631,23,'scroll_of_dia','dia',1,1676,28,0,29);
 INSERT INTO `item_basic` VALUES (4632,24,'scroll_of_dia_ii','dia_ii',1,1676,28,0,1556);
+INSERT INTO `item_basic` VALUES (4633,26,'scroll_of_dia_iii','dia_iii',1,1676,28,0,10000);
```
Comment (nfaletra) [inline sql/item_basic.sql]: subid 25

File: sql/item_basic.sql [hunk 10]
```diff
@@ -6162,6 +6177,9 @@ INSERT INTO `item_basic` VALUES (6565,0,'persikos_snow_cone','per._snow_cone',12
 INSERT INTO `item_basic` VALUES (6566,0,'bonanza_biscuit','bonanza_biscuit',99,30296,0,1,0);
 INSERT INTO `item_basic` VALUES (6567,0,'tropical_crepe','tropical_crepe',12,1548,57,0,0);
 INSERT INTO `item_basic` VALUES (6568,0,'crepe_des_rois','crepe_des_rois',12,1548,57,0,0);
+INSERT INTO `item_basic` VALUES (6569,57,'scroll_of_slow_ii','slow_ii',1,1676,28,0,1000);
```
Comment (nfaletra) [inline sql/item_basic.sql]: subid 79

File: sql/item_basic.sql [hunk 11]
```diff
@@ -6162,6 +6177,9 @@ INSERT INTO `item_basic` VALUES (6565,0,'persikos_snow_cone','per._snow_cone',12
...
+INSERT INTO `item_basic` VALUES (6570,59,'scroll_of_paralyze_ii','paralyze_ii',1,1676,28,0,10000);
```
Comment (nfaletra) [inline sql/item_basic.sql]: subid 80

File: scripts/globals/items/scroll_of_suiton_san.lua (hunk 6 above)
Comment (nfaletra) [reply scripts/globals/items/scroll_of_suiton_san.lua]: Still waiting on this one, but everything else looks good! :+1:

**Comment by m241dan** [comment]
Boom

**Pull request merged**